src/test/skript/tests/syntaxes/conditions/CondIsCharged.sk

```diff
@@ -0,0 +1,31 @@
+test "is charged":
+	spawn a creeper at (spawn of world "world"):
```

Comment:
You can also use event-location in the case that we change the location for some reason.